outputs.tf

```diff
@@ -14,13 +14,42 @@
  * limitations under the License.
  */
 
-output "names_and_display_names" {
-  description = "Map of name => folder resource name."
-  value       = zipmap(var.names, google_folder.folders.*.display_name)
+output "folder" {
+  description = "Folder resource (for single use)."
+  value       = google_folder.folders[0]
 }
 
-output "names_and_ids" {
```

Comment:
I'm not sure we want to make this a breaking change by renaming the outputs?

If we do please include a short migration guide.

Reply:
Added migration guide for changed/new outputs, and parent variable change.